### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by cake03** [transmission/transmission#1925] [General]
Transmission has over 159 commits since 22/05/2020, and an update is needed because most people would prefer downloading a pre-compiled release over `git clone`ing it

**Comment by derpyjoystick** [comment]
This is honestly the largest pain point of this project. I really do get all the participants code this for free. Something I think is both commendable and not rewarded enough, but the release schedule seems incredibly (and arbitrarily) restrictive.

I know pre-compiled nightly versions exist, but perhaps it may be time to get on a more frequent release schedule so that we don't see incremental version bumps that literally take years when the codebase is there, waiting.

It all just seems unnecessary restrictive ☹️

**Comment by Kurville** [comment]
And make the release a universal binary (Intel and M1) for macOS... Please!

**Comment by pleasejustgo** [comment]
None of the devs are responding. They probably don't care. Can someone at least reply?

**Comment by trypie** [comment]
I think Transmission is dead - unless we can get some sort of reply here?
Even if we had a tutorial how to transfer everything to a new torrent app that would at least give us something

**Comment by Kurville** [comment]
> I think Transmission is dead

As far as the macOS is concerned, there's regular activity here https://build.transmissionbt.com/job/trunk-mac/. It's just not Apple Silicon yet, although Rosetta is doing a good job at running it.

**Reply by grr** [reply]
> > I think Transmission is dead
> 
> As far as the macOS is concerned, there's regular activity here https://build.transmissionbt.com/job/trunk-mac/. It's just not Apple Silicon yet, although Rosetta is doing a good job at running it.

I was using those builds a few months back, but the update feature would never work, so I'd have to manually check and download new versions.

**Reply by Kurville** [reply]
Transmission update check is part of my daily routine now... ;)

**Comment by ralyodio** [comment]
Need something for ubuntu server for a seedbox

**Comment by ckerr** [comment]
There are three different "new release" discussions right now, so I'm locking two of them :smile:
